src/Servers/IIS/AspNetCoreModuleV2/OutOfProcessRequestHandler/serverprocess.h

```diff
@@ -6,7 +6,10 @@
 #include <random>
 #include <map>
 
-#define MIN_PORT                                    1025
+// This value is somewhat arbitrary. Fixed-port services tend to
+// use ports near the minimum (1025) so this number is much higher
+// to reduce the chance of collisions.
+#define MIN_PORT                                    10000
```

Comment:
An issue I see with this change is that we are now unnecessarily stopping someone from choosing a port lower than 10000 via the `ASPNETCORE_PORT` environment variable
https://github.com/dotnet/aspnetcore/blob/ac542f4916b26fde739fd146511be70898af233a/src/Servers/IIS/AspNetCoreModuleV2/OutOfProcessRequestHandler/serverprocess.cpp#L159-L165

Reply:
Can we just remove this constraint? `if (m_dwPort >MAX_PORT || m_dwPort < MIN_PORT) `

Reply:
No, we should probably change the 10000 value to MIN_PORT_RANDOM or something and keep MIN_PORT at 1025 which is used for this check.

Reply:
I have updated it according to your suggestion. Please take another look. Thank you!!